Tutorials/python/Simple_Firewall/Add_filter_Rule_add_rule.py

```diff
@@ -19,6 +19,8 @@
   # extract the database ID
   device_db_id = device['id'][-3:]
 
+  if context['icmp'] == 'true':
```

Comment:
that's ok, I don't want to block this merge
just FYI python one line statement for such cases 
`if <expr>: <statement>`

Reply:
another option - use Ternary Operator